sequencer_sabotage_simulation/utils.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from sequencer_sabotage_simulation.schema import (NodeSimulationStateItem,
+                                                  NodeSabotageSimulationState,
+                                                  NetworkData)
+
+
+def get_node_sabotage_config(config_file_path: str, address: str) -> NodeSabotageSimulationState:
+    return _parse_simulation_state(config_file_path)[address]
+
+
+def _validate_item(item: dict) -> NodeSimulationStateItem:
+    if not isinstance(item, dict):
+        raise ValueError("Item must be a dictionary")
+    if "time_duration" not in item or not isinstance(item["time_duration"], (int, float)):
```

Comment:
Duration means "the time during which something continue" so `time` in `duration_time` is redundant and `duration` alone is a better